src/index.ts

```diff
@@ -2,27 +2,33 @@ import * as engine from './engine';
 import { getPageFeatures } from './features';
 import { viewStore, matchedAudienceStore } from './store';
 import { timeStampInSecs } from './utils';
-import { hasGdprConsent } from './gdpr';
+import { waitOnConsent } from './gdpr';
 import {
   PageFeatureGetter,
   MatchedAudience,
   AudienceDefinition,
 } from '../types';
 
+export { runOnConsent, waitOnConsent } from './gdpr';
+
 interface Config {
   pageFeatureGetters: PageFeatureGetter[];
   audienceDefinitions: AudienceDefinition[];
   vendorIds?: number[];
   omitGdprConsent?: boolean;
+  allowMultipleRuns?: boolean; // For testing purposes only
```

Comment:
I am not sure I get why this is needed, even for testing?

Reply:
Yeah it was for `hasRun` but I don't think it's needed anymore